contracts/interfaces/IFractalUsul.sol

```diff
@@ -2,44 +2,166 @@
 pragma solidity ^0.8.0;
 
 import "@gnosis.pm/safe-contracts/contracts/common/Enum.sol";
+import "../FractalUsul.sol";
 
 interface IFractalUsul {
-    /// @dev Returns the hash of a transaction in a proposal.
-    /// @param proposalId the proposal to inspect.
-    /// @param index the transaction to inspect.
-    /// @return transaction hash.
-    function getTxHash(uint256 proposalId, uint256 index)
-        external
-        view
-        returns (bytes32);
+    enum ProposalState {
+        Active,
+        Canceled,
+        Timelocked,
+        Executed,
+        Executable,
+        Uninitialized
```

Comment:
Enum values are generally written as all caps, we could do that here now that we're cleaning it up